fdbserver/KeyValueStoreRocksDB.actor.cpp

```diff
@@ -38,18 +39,21 @@ struct RocksDBKeyValueStore : IKeyValueStore {
 	using CF = rocksdb::ColumnFamilyHandle*;
 
 	struct Writer : IThreadPoolReceiver {
-		DB& db;
-		UID id;
```

Comment:
Accessing `store-db` directly had the downside of making `UID id` (and the new proc parameter) to the constructor redundant.  Now, I pass them implicitly, and `store` is turning into a "god object."

Suggestion's welcome.

Reply:
In the end, I guess we either have to have a god object or a `**` one way or another. I've just gotten used to the weirdness of a `*&`, so I'm a little biased towards that vs the god object. This file is small enough (and likely to remain small enough) that the downsides of a god object are substantially reduced, so I'm fine with this change.